### DISCUSSION TITLE
Skia with a bug at release 3.0.4 ?

### DISCUSSION THREAD

**Discussion by serginh023** [ammariqais/SkiaForUnity#48] [General]
I `ve been downloaded the 3.0.4 and the error message of "Unsafe" class not reconized is appearing at Unity console, someone knows what is that?
(I think about a bug in Unsafe .nupkg)
Tested in Unity 2022.2.11.

![image](https://github.com/ammariqais/SkiaForUnity/assets/13037064/a374fb8a-6aca-40b9-a469-0d78bec738ee)
![image](https://github.com/ammariqais/SkiaForUnity/assets/13037064/3c99b527-c3f0-48d6-a582-f64629604997)

**Comment by ammariqais** [comment]
please make sure to added external lib Unsafe, you can download it from here

https://github.com/ammariqais/SkiaForUnity/tree/main/SkiaUnity/Assets/SkiaSharp/Dependencies
